drivers/filesystems/ext2/src/read.c

```diff
@@ -285,7 +285,7 @@ Ext2ReadVolume (IN PEXT2_IRP_CONTEXT IrpContext)
                         }
                     }
 
-                    Ext2CompleteIrpContext(IrpContext, Status);;
+                    Ext2CompleteIrpContext(IrpContext, Status);
```

Comment:
3rd party code?

Reply:
Do you want me to revert this one ? Or do we take it as it won't harm ?
I only took out code in 3rdparty folder

Reply:
> 3rd party code?

Yes: see https://git.reactos.org/?p=reactos.git;a=blob;f=media/doc/README.FSD;hb=HEAD

> revert

Options are:
1. Cherry-pick from upstream, if already fixed there.
2. Use `__REACTOS__`, though a diff for such a minor change may be refused...
3. Disable warning (CMakeLists.txt) for the time being, if that applies.
4. Leave it as is :-/

Reply:
Revert...